app/src/app/[lng]/[inventory]/preferences/transportation/TransportationPage.tsx

```diff
@@ -0,0 +1,28 @@
+"use client";
+
+import { Heading, SimpleGrid, Text, VStack } from "@chakra-ui/react";
+import type { TFunction } from "i18next";
+import { PreferenceCard } from "@/app/[lng]/[inventory]/preferences/activities/PreferenceCard";
+import { TRANSPORTATION_ITEMS } from "@/app/[lng]/[inventory]/preferences/transportation/TRANSPORTATION_ITEMS";
+
+export default function TransportationPage({ t }: { t: TFunction }) {
+  function onClick() {}
+
+  return (
+    <VStack marginLeft={"5vw"}>
+      <Heading my={"24px"}>{t("which-transportation")}</Heading>
```

Comment:
You can also use the Chakra units for brevity, in most cases it's 1 = 4px. Optional of course though.
```suggestion
      <Heading my={6}>{t("which-transportation")}</Heading>
```